lib/perl/Genome/VariantReporting/Framework/Component/Report/MergedReport.pm

```diff
@@ -540,5 +541,25 @@ sub get_entry_source {
     die sprintf("No entry source for report (%s)", $report);
 }
 
+sub category {
+    my $self = shift;
+
+    my @report_users = map { $_->users('label like' => 'report:%') } $self->report_results;
+    my $category;
+    for my $user (@report_users) {
+        if ($user->label =~ /report:(.*)/) {
+            my $metadata_json = $1;
+            my $m = from_json($metadata_json);
```

Comment:
I'm late to the game, but it seems like JSON would be better stored elsewhere (a SoftwareResult::Metric perhaps?) than in a user label.